### ISSUE TITLE
NameError: name 'get_logger' is not defined

### ISSUE THREAD

**Issue by Jamie-Chang** [Kludex/uvicorn#285]
Happens only when running in debug mode.
```
$ uvicorn app:App --debug
Traceback (most recent call last):
  File "/home/jamie.chang/.local/bin/uvicorn", line 11, in <module>
    sys.exit(main())
  File "/home/jamie.chang/.local/lib/python3.5/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/home/jamie.chang/.local/lib/python3.5/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/home/jamie.chang/.local/lib/python3.5/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/jamie.chang/.local/lib/python3.5/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/home/jamie.chang/.local/lib/python3.5/site-packages/uvicorn/main.py", line 166, in main
    logger = get_logger(log_level)
NameError: name 'get_logger' is not defined```

**Comment by lovelydinosaur** [comment]
Released as 0.3.30. Planning to work on bumping the test coverage up in order to avoid this kind of breakage in the future.

**Comment by Jamie-Chang** [comment]
That was fast, thanks!
